app/javascript/elm/src/Tooltip.elm

```diff
@@ -0,0 +1,71 @@
+module Tooltip exposing
+    ( TooltipText
+    , crowdMap
+    , fixedTab
+    , locationFilter
+    , mobileTab
+    , parameterFilter
+    , profilesFilter
+    , sensorFilter
+    , streamingToggleFilter
+    , tagsFilter
+    , timeRangeFilter
+    , typeToggleFilter
+    , view
+    )
+
+import Html exposing (Html, span, text)
+import Html.Attributes exposing (attribute)
+
+
+type TooltipText
+    = TooltipText String
+
+
+view : TooltipText -> Html msg
+view (TooltipText tooltipText) =
+    span [ attribute "data-tippy-content" tooltipText ] [ text "?" ]
```

Comment:
Awesome! I really like that structure with the opaque type, I'll try to use them more :)

Reply:
Yeah let's get deep into types!